projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.ts

```diff
@@ -285,14 +287,15 @@ export class IgxSimpleComboComponent extends IgxComboBaseDirective implements Co
         if (this.collapsed && this.comboInput.focused) {
             this.open();
         }
-        if (!this.comboInput.value.trim() && super.selection.length) {
+        if (!this.comboInput.value.trim() && super.selection.length && super.selection[0][this.valueKey] !== "") {
```

Comment:
This
```ts
super.selection[0][this.valueKey] !== ""
```
will return always `true` if there is no `valueKey`. Also this will return `true` is the selected item value is not empty string. This will return `false` only if there is `valueKey` and the value of the selected item is empty string. Do we need this change?

Reply:
one more thing. This entire `if` statement is trying to check if user initially pressed space key. If so, it clears the and actually suppress the filtering. If user needs to filter all the items containing empty space, he should press the space bar twice - variation of the issue related to this PR.

Reply:
@jackofdiamond5 
Please take a look at this PR.

Reply:
The way I see it, we don't really need this line: https://github.com/IgniteUI/igniteui-angular/blob/668f38b2345e978e709d36bd1099d0d43c6f2e21/projects/igniteui-angular/src/lib/simple-combo/simple-combo.component.ts#L243

Since on opening we need to know what is being filtered and what the current search term is in order to show the respective elements in the dropdown. So imo, just get rid of that line altogether and this should fix the issue. The reason why it was present to begin with was because it was a partial fix of an issue related to entering partial matches and making sure that the combo clears on blur if no match is found. With the current implementation, the combo does not need that assignment anymore.

On a side note, I found another thing which is kind of related to filtering - https://github.com/IgniteUI/igniteui-angular/issues/14352 which can be fixed relatively easy, if we assume that we don't want to apply selection and we just want to close the combo on  `ArrowUp`. But this can be considered a UX question, so I've logged a separate issue just in case. P.S. I've tested with the WC combo and it immediately closes on `ArrowUp` sooo yeah.